__tests__/app.test.js

```diff
@@ -353,4 +353,40 @@ test("404: Responds with error message if article does not exist", () => {
     expect(body.msg).toBe("Article Doesn't Exist")
   })
 })
+})
+describe("DELETE /api/comments/:comments_id", () => {
+  test("204: Should delete the comment with the given ID", () => {
+    return request(app)
+    .delete("/api/comments/1")
+    .expect(204)
+    .then(() => {
+      return db.query(`SELECT * FROM comments WHERE comment_id = $1`, [1]).then(({ rows }) => {
+        expect(rows.length).toBe(0)
+      })
+    })
+  })
```

Comment:
This is a really nice first test, and I like the idea of querying the db here to do this check, but this test combined with the below error handling should already give you the confidence that it is doing what we expect as well